### ISSUE TITLE
Blood Magic Upgrades? Cant seem to get them to work

### ISSUE THREAD

**Issue by Bobynoby** [Ipsis/Woot#130]
I have tried for so long to get life essence out of the Woot farm, I just cant for some reason. Using Woot 1.2.3 and Bloodmagic 1.1.0-69

I have a max tier Woot farm, With tier 3 blood magic upgrades but no matter what tank I put next to the factory or the proxy, It doesn't output anything. The farm is going as it is receiving power and killing zombies. I tried placing a blood altar next to the factory as well but that didn't work either.

I must be doing something wrong somewhere, as I tried the mods just by them selves and I couldn't get it to work.

**Comment by Bobynoby** [comment]
Nevermind, I was screwing up something on my end...

**Comment by Ipsis** [comment]
No problem. Was it something that was not clear in my mod? Can I update the documentation?

**Comment by Bobynoby** [comment]
I thought I was able to use it as a kind of easy way to start out blood magic by pumping straight from the factory into a tank, But I didn't know that I needed to make a Ritual of the infernal machine to get blood out of it. It was hard to find that out so maybe you might need to add it to the in game book? And update the github wiki to have info about the  blood magic integration as I couldn't seem to find anything there as well.
